opus_codec_registration.src.html

```diff
@@ -151,6 +159,31 @@
   <dd>
     Configures the frame duration, in milliseconds, of output {{EncodedAudioChunk}}s.
   </dd>
+  <dt><dfn dict-member for=OpusEncoderConfig>complexity</dfn></dt>
+  <dd>
+    Configures the encoder's computational complexity, as described 2.1.9. of
+    [[RFC6716]]. The valid range is `0` to `10`, with `10` representing the highest
+    complexity. If no value is specificied, the default value is platform-specific:
+    User Agents <em class="rfc2119">SHOULD</em> set a default of `5` for mobile
+    platforms, and a default of `9` for all other platforms.
+  </dd>
+  <dt><dfn dict-member for=OpusEncoderConfig>packetlossperc</dfn></dt>
+  <dd>
+    Configures the encoder's expected packet loss percentage. The valid range is
+    `0` to `100`.
+
+    NOTE: The packet loss percentage might be updated over the course of the
+    encoding, and it is recommended for User Agents to support these reconfigurations.
+  </dd>
+  <dt><dfn dict-member for=OpusEncoderConfig>useinbandfec</dfn></dt>
+  <dd>
+    Specifies whether the encoder provides Opus in-band FEC, as described by
```

Comment:
Worth spelling out Forward Error Correction (FEC) and whatever DTX stands for?

Reply:
Done.